Makefile

```diff
@@ -375,6 +411,13 @@ internal-clean: $$(KIND)
 internal-distclean: $$(KIND)
 	@rm -rf ./_build/$(KIND)
 
+prod-deb-package:
+	export DEBEMAIL=$(AE_DEB_EMAIL); \
```

Comment:
Could you please silent the exports?

Example output (too noisy):
```
export DEBEMAIL="info@aeternity.com"; \
export DEBFULLNAME="Aeternity Team" ;\
dch -v `cat VERSION` "Release notes are available in /usr/share/doc/aeternity-node/docs/release-notes/RELEASE-NOTES-`cat VERSION`.md"; \
dch -r "Release notes are available in /usr/share/doc/aeternity-node/docs/release-notes/RELEASE-NOTES-`cat VERSION`.md"; \
debuild -uc -us

```

Reply:
Just to clarify, you want me to suppress the lines from the example, not the output of the build?

Example:
```
	@export DEBEMAIL=$(AE_DEB_EMAIL); \
...
```

This will only the lines in question, not the build output.

Reply:
The makefile commands "echo" for export commands.

Reply:
And maybe also for `dch` commands

Reply:
Addressed in e4a9f0a. After 22c1d48 addressed there.